### PR TITLE
Irminger 11 precruise deploy csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Irminger precruise deploy csvs; can verify via config sheets in Vault.  
Placeholder lat/lon, depths taken from Site Char Table 4 - attached.
[3202-00007_Table4.pdf](https://github.com/WHOIGit/ooicgsn-asset-management/files/15349792/3202-00007_Table4.pdf)

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[Irminger 11 AM Updates](https://trello.com/c/jAVA6WUL/237-irminger-11-am-updates)

File: deployment/GI01SUMO_Deploy.csv [hunk 1]
```diff
@@ -629,3 +629,67 @@ AR76-03,,,,GI01SUMO-SBD12-06-METBKA005,10,1,2023-08-31T16:00:00,,CGMGI-01SUMO-00
 AR76-03,,,,GI01SUMO-SBD12-06-METBKA006,10,1,2023-08-31T16:00:00,,CGMGI-01SUMO-00010,,CGINS-METSWR-00253,59.944783,-39.5736,,-5,2663,,,C
 AR76-03,,,,GI01SUMO-SBD12-06-METBKA007,10,1,2023-08-31T16:00:00,,CGMGI-01SUMO-00010,,CGINS-METWND-00258,59.944783,-39.5736,,-6,2663,sensor raised to same height as FDCHP,,C
 AR76-03,,,,GI01SUMO-SBD12-08-FDCHPA000,10,1,2023-08-31T16:00:00,,CGMGI-01SUMO-00010,,CGINS-FDCHPB-03904,59.944783,-39.5736,,-6,2663,,,C
+AR84-01,,,,GI01SUMO-RID16-00-DCLENG000,11,1,2024-06-02T00:00:00,,CGMGI-01SUMO-00011,,CGCON-NDCL16-00006,59.9344,-39.4685,,12,2691,,,D
```
Comment (cooleyky) [inline deployment/GI01SUMO_Deploy.csv:632]: S/N changed to -00045 on 5/21/2024

File: deployment/GI01SUMO_Deploy.csv [hunk 2]
```diff
@@ -629,3 +629,67 @@ AR76-03,,,,GI01SUMO-SBD12-06-METBKA005,10,1,2023-08-31T16:00:00,,CGMGI-01SUMO-00
...
+AR84-01,,,,GI01SUMO-RII11-02-FLORDG031,11,1,2024-06-02T00:00:00,,CGMGI-01SUMO-00011,,CGINS-FLORDG-03969,59.9344,-39.4685,,40,2691,,,D
+AR84-01,,,,GI01SUMO-RII11-02-FLORDG032,11,1,2024-06-02T00:00:00,,CGMGI-01SUMO-00011,,CGINS-FLORDG-03292,59.9344,-39.4685,,80,2691,,,D
+AR84-01,,,,GI01SUMO-RII11-02-FLORDG033,11,1,2024-06-02T00:00:00,,CGMGI-01SUMO-00011,,CGINS-FLORDG-03970,59.9344,-39.4685,,130,2691,,,D
+AR84-01,,,,GI01SUMO-RII11-02-PCO2WC051,11,1,2024-06-02T00:00:00,,CGMGI-01SUMO-00011,,CGINS-PCO2WC-C0107,59.9344,-39.4685,,40,2691,,,D
+AR84-01,,,,GI01SUMO-RII11-02-PCO2WC052,11,1,2024-06-02T00:00:00,,CGMGI-01SUMO-00011,,CGINS-PCO2WC-C0120,59.9344,-39.4685,,80,2691,,,D
+AR84-01,,,,GI01SUMO-RII11-02-PCO2WC053,11,1,2024-06-02T00:00:00,,CGMGI-01SUMO-00011,,CGINS-PCO2WC-C0140,59.9344,-39.4685,,130,2691,,,D
+AR84-01,,,,GI01SUMO-RII11-02-PHSENE041,11,1,2024-06-02T00:00:00,,CGMGI-01SUMO-00011,,CGINS-PHSENE-P0138,59.9344,-39.4685,,20,2691,,,D
+AR84-01,,,,GI01SUMO-RII11-02-PHSENE042,11,1,2024-06-02T00:00:00,,CGMGI-01SUMO-00011,,CGINS-PHSENE-P0140,59.9344,-39.4685,,100,2691,,,D
```
Comment (cooleyky) [inline deployment/GI01SUMO_Deploy.csv:665]: Both PHSEN-Es also had S/N changes: P0138-->P0172, P0140-->P0177

File: deployment/GI01SUMO_Deploy.csv [hunk 3]
```diff
@@ -629,3 +629,67 @@ AR76-03,,,,GI01SUMO-SBD12-06-METBKA005,10,1,2023-08-31T16:00:00,,CGMGI-01SUMO-00
...
+AR84-01,,,,GI01SUMO-SBD11-06-METBKA004,11,1,2024-06-02T00:00:00,,CGMGI-01SUMO-00011,,CGINS-METLWR-00295,59.9344,-39.4685,,-5,2691,,,D
+AR84-01,,,,GI01SUMO-SBD11-06-METBKA005,11,1,2024-06-02T00:00:00,,CGMGI-01SUMO-00011,,CGINS-METPRC-00246,59.9344,-39.4685,,-5,2691,,,D
+AR84-01,,,,GI01SUMO-SBD11-06-METBKA006,11,1,2024-06-02T00:00:00,,CGMGI-01SUMO-00011,,CGINS-METSWR-00294,59.9344,-39.4685,,-5,2691,,,D
+AR84-01,,,,GI01SUMO-SBD11-06-METBKA007,11,1,2024-06-02T00:00:00,,CGMGI-01SUMO-00011,,CGINS-METWND-00253,59.9344,-39.4685,,-6,2691,,,D
+AR84-01,,,,GI01SUMO-SBD11-08-NUTNRB000,11,1,2024-06-02T00:00:00,,CGMGI-01SUMO-00011,,CGINS-NUTNRB-01062,59.9344,-39.4685,,1,2691,,,D
+AR84-01,,,,GI01SUMO-SBD12-00-DCLENG000,11,1,2024-06-02T00:00:00,,CGMGI-01SUMO-00011,,CGCON-BDCL12-50010,59.9344,-39.4685,,0,2691,,,D
+AR84-01,,,,GI01SUMO-SBD12-01-OPTAAD000,11,1,2024-06-02T00:00:00,,CGMGI-01SUMO-00011,,CGINS-OPTAAD-00151,59.9344,-39.4685,,1,2691,,,D
+AR84-01,,,,GI01SUMO-SBD12-02-FLORTD000,11,1,2024-06-02T00:00:00,,CGMGI-01SUMO-00011,,CGINS-FLORTD-01323,59.9344,-39.4685,,1,2691,,,D
```
Comment (cooleyky) [inline deployment/GI01SUMO_Deploy.csv:683]: S/N changed from 1323 to 1324

**Review by cooleyky — COMMENTED** [review]
Changes needed on GI01SUMO for updated serial numbers

File: deployment/GI01SUMO_Deploy.csv [hunk 4]
```diff
@@ -680,16 +680,16 @@ AR84-01,,,,GI01SUMO-SBD11-06-METBKA007,11,1,2024-06-02T00:00:00,,CGMGI-01SUMO-00
...
 AR84-01,,,,GI01SUMO-SBD12-06-METBKA001,11,1,2024-06-02T00:00:00,,CGMGI-01SUMO-00011,,CGINS-METCTD-11560,59.9344,-39.4685,,1,2691,,,D
 AR84-01,,,,GI01SUMO-SBD12-06-METBKA002,11,1,2024-06-02T00:00:00,,CGMGI-01SUMO-00011,,CGINS-METBPR-00266,59.9344,-39.4685,,-5,2691,,,D
 AR84-01,,,,GI01SUMO-SBD12-06-METBKA003,11,1,2024-06-02T00:00:00,,CGMGI-01SUMO-00011,,CGINS-METHRH-00279,59.9344,-39.4685,,-5,2691,,,D
-#AR84-01,,,,GI01SUMO-SBD12-06-METBKA004,11,1,2024-06-02T00:00:00,,CGMGI-01SUMO-00011,,,59.9344,-39.4685,,-5,2691,not allocated yet,,D
+AR84-01,,,,GI01SUMO-SBD12-06-METBKA004,11,1,2024-06-02T00:00:00,,CGMGI-01SUMO-00011,,CGINS-METLWR-00284,59.9344,-39.4685,,-5,2691,,,D
 AR84-01,,,,GI01SUMO-SBD12-06-METBKA005,11,1,2024-06-02T00:00:00,,CGMGI-01SUMO-00011,,CGINS-METPRC-00255,59.9344,-39.4685,,-5,2691,,,D
-#AR84-01,,,,GI01SUMO-SBD12-06-METBKA006,11,1,2024-06-02T00:00:00,,CGMGI-01SUMO-00011,,,59.9344,-39.4685,,-5,2691,not allocated yet,,D
+AR84-01,,,,GI01SUMO-SBD12-06-METBKA006,11,1,2024-06-02T00:00:00,,CGMGI-01SUMO-00011,,CGINS-METSWR-00278,59.9344,-39.4685,,-5,2691,,,D
```
Comment (cooleyky) [inline deployment/GI01SUMO_Deploy.csv:693]: All updates look good!

**Review by cooleyky — APPROVED** [review]
looks good

File: deployment/GI01SUMO_Deploy.csv [hunk 5]
```diff
@@ -629,3 +629,67 @@ AR76-03,,,,GI01SUMO-SBD12-06-METBKA005,10,1,2023-08-31T16:00:00,,CGMGI-01SUMO-00
...
+AR84-01,,,,GI01SUMO-SBD12-05-WAVSSA000,11,1,2024-06-02T00:00:00,,CGMGI-01SUMO-00011,,CGINS-WAVSSA-05791,59.9344,-39.4685,,0,2691,,,D
+AR84-01,,,,GI01SUMO-SBD12-06-METBKA000,11,1,2024-06-02T00:00:00,,CGMGI-01SUMO-00011,,CGINS-METLGR-00007,59.9344,-39.4685,,0,2691,,,D
+AR84-01,,,,GI01SUMO-SBD12-06-METBKA001,11,1,2024-06-02T00:00:00,,CGMGI-01SUMO-00011,,CGINS-METCTD-11560,59.9344,-39.4685,,1,2691,,,D
+AR84-01,,,,GI01SUMO-SBD12-06-METBKA002,11,1,2024-06-02T00:00:00,,CGMGI-01SUMO-00011,,CGINS-METBPR-00266,59.9344,-39.4685,,-5,2691,,,D
+AR84-01,,,,GI01SUMO-SBD12-06-METBKA003,11,1,2024-06-02T00:00:00,,CGMGI-01SUMO-00011,,CGINS-METHRH-00279,59.9344,-39.4685,,-5,2691,,,D
+AR84-01,,,,GI01SUMO-SBD12-06-METBKA004,11,1,2024-06-02T00:00:00,,CGMGI-01SUMO-00011,,CGINS-METLWR-00284,59.9344,-39.4685,,-5,2691,,,D
+AR84-01,,,,GI01SUMO-SBD12-06-METBKA005,11,1,2024-06-02T00:00:00,,CGMGI-01SUMO-00011,,CGINS-METPRC-00255,59.9344,-39.4685,,-5,2691,,,D
+AR84-01,,,,GI01SUMO-SBD12-06-METBKA006,11,1,2024-06-02T00:00:00,,CGMGI-01SUMO-00011,,CGINS-METSWR-00278,59.9344,-39.4685,,-5,2691,,,D
```
Comment (cdobs) [inline deployment/GI01SUMO_Deploy.csv:693]: looks good

File: deployment/GI02HYPM_Deploy.csv [hunk 6]
```diff
@@ -85,3 +85,12 @@ AR76-03,,,,GI02HYPM-WFP02-04-CTDPFL000,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00
...
+AR84-01,,,,GI02HYPM-WFP02-00-WFPENG000,11,1,2024-06-02T00:00:00,,CGMGI-02HYPM-00011,CGVEH-GWFPGX-10401,CGCON-GWFPGX-10401,59.9704,-39.4888,,2560,2678,,,E
+AR84-01,,,,GI02HYPM-WFP02-01-FLORDL000,11,1,2024-06-02T00:00:00,,CGMGI-02HYPM-00011,CGVEH-GWFPGX-10401,CGINS-FLORDL-03280,59.9704,-39.4888,,2560,2678,,,E
+AR84-01,,,,GI02HYPM-WFP02-03-DOSTAL000,11,1,2024-06-02T00:00:00,,CGMGI-02HYPM-00011,CGVEH-GWFPGX-10401,CGINS-DOSTAL-01476,59.9704,-39.4888,,2560,2678,,,E
+AR84-01,,,,GI02HYPM-WFP02-04-CTDPFL000,11,1,2024-06-02T00:00:00,,CGMGI-02HYPM-00011,CGVEH-GWFPGX-10401,CGINS-CTDPFL-00131,59.9704,-39.4888,,2560,2678,,,E
+AR84-01,,,,GI02HYPM-WFP02-05-VEL3DL000,11,1,2024-06-02T00:00:00,,CGMGI-02HYPM-00011,CGVEH-GWFPGX-10401,CGINS-VEL3DL-01109,59.9704,-39.4888,,2560,2678,,,E
+AR84-01,,,,GI02HYPM-MPM01-02-ZPLSGA009,11,1,2024-06-02T00:00:00,,CGMGI-02HYPM-00011,,CGINS-ZPLSGA-55107,59.9704,-39.4888,,150,2678,upward-looking,,E
+AR84-01,,,,GI02HYPM-MPM01-02-ZPLSGA010,11,1,2024-06-02T00:00:00,,CGMGI-02HYPM-00011,,CGINS-ZPLSGA-55109,59.9704,-39.4888,,150,2678,downward-looking,,E
```
Comment (cdobs) [inline deployment/GI02HYPM_Deploy.csv:96]: looks good

File: deployment/GI03FLMA_Deploy.csv [hunk 7]
```diff
@@ -259,3 +259,29 @@ AR76-03,,,,GI03FLMA-RIS01-00-SIOENG000,10,1,2023-09-05T14:56:00,,CGMGI-03FLMA-00
...
+AR84-01,,,,GI03FLMA-RIM01-02-CTDMOG048,11,1,2024-06-02T00:00:00,,CGMGI-03FLMA-00011,,CGINS-CTDMOG-12375,59.7674,-39.8426,,500,2700,,,D
+AR84-01,,,,GI03FLMA-RIM01-02-CTDMOH049,11,1,2024-06-02T00:00:00,,CGMGI-03FLMA-00011,,CGINS-CTDMOH-13446,59.7674,-39.8426,,750,2700,,,D
+AR84-01,,,,GI03FLMA-RIM01-02-CTDMOH050,11,1,2024-06-02T00:00:00,,CGMGI-03FLMA-00011,,CGINS-CTDMOH-11704,59.7674,-39.8426,,1000,2700,,,D
+AR84-01,,,,GI03FLMA-RIM01-02-CTDMOH051,11,1,2024-06-02T00:00:00,,CGMGI-03FLMA-00011,,CGINS-CTDMOH-13653,59.7674,-39.8426,,1500,2700,,,D
+AR84-01,,,,GI03FLMA-RIS01-00-SIOENG000,11,1,2024-06-02T00:00:00,,CGMGI-03FLMA-00011,,CGCON-SIOCON-13284,59.7674,-39.8426,,30,2700,,,D
+AR84-01,,,,GI03FLMA-RIS01-03-DOSTAD000,11,1,2024-06-02T00:00:00,,CGMGI-03FLMA-00011,,CGINS-DOSTAD-00439,59.7674,-39.8426,,30,2700,,,D
+AR84-01,,,,GI03FLMA-RIS01-04-PHSENF000,11,1,2024-06-02T00:00:00,,CGMGI-03FLMA-00011,,CGINS-PHSENF-P0173,59.7674,-39.8426,,30,2700,,,D
+AR84-01,,,,GI03FLMA-RIS01-05-FLORTD000,11,1,2024-06-02T00:00:00,,CGMGI-03FLMA-00011,,CGINS-FLORTD-00994,59.7674,-39.8426,,30,2700,,,D
```
Comment (cdobs) [inline deployment/GI03FLMA_Deploy.csv:287]: looks good

File: deployment/GI03FLMB_Deploy.csv [hunk 8]
```diff
@@ -259,3 +259,29 @@ AR76-03,,,,GI03FLMB-RIS01-00-SIOENG000,10,1,2023-09-06T15:53:00,,CGMGI-03FLMB-00
...
+AR84-01,,,,GI03FLMB-RIM01-02-CTDMOG068,11,1,2024-06-02T00:00:00,,CGMGI-03FLMB-00011,,CGINS-CTDMOG-12376,59.7147,-39.3168,,500,2830,,,D
+AR84-01,,,,GI03FLMB-RIM01-02-CTDMOH069,11,1,2024-06-02T00:00:00,,CGMGI-03FLMB-00011,,CGINS-CTDMOH-10264,59.7147,-39.3168,,750,2830,,,D
+AR84-01,,,,GI03FLMB-RIM01-02-CTDMOH070,11,1,2024-06-02T00:00:00,,CGMGI-03FLMB-00011,,CGINS-CTDMOH-13445,59.7147,-39.3168,,1000,2830,,,D
+AR84-01,,,,GI03FLMB-RIM01-02-CTDMOH071,11,1,2024-06-02T00:00:00,,CGMGI-03FLMB-00011,,CGINS-CTDMOH-13646,59.7147,-39.3168,,1500,2830,,,D
+AR84-01,,,,GI03FLMB-RIS01-00-SIOENG000,11,1,2024-06-02T00:00:00,,CGMGI-03FLMB-00011,,CGCON-SIOCON-13987,59.7147,-39.3168,,30,2830,,,D
+AR84-01,,,,GI03FLMB-RIS01-03-DOSTAD000,11,1,2024-06-02T00:00:00,,CGMGI-03FLMB-00011,,CGINS-DOSTAD-00457,59.7147,-39.3168,,30,2830,,,D
+AR84-01,,,,GI03FLMB-RIS01-04-PHSENF000,11,1,2024-06-02T00:00:00,,CGMGI-03FLMB-00011,,CGINS-PHSENF-P0174,59.7147,-39.3168,,30,2830,,,D
+AR84-01,,,,GI03FLMB-RIS01-05-FLORTD000,11,1,2024-06-02T00:00:00,,CGMGI-03FLMB-00011,,CGINS-FLORTD-01122,59.7147,-39.3168,,30,2830,,,D
```
Comment (cdobs) [inline deployment/GI03FLMB_Deploy.csv:287]: looks good

**Review by cdobs — APPROVED** [review]
looks good

**Pull request merged**